src/dispatchers/touchDispatcher.ts

```diff
@@ -0,0 +1,138 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export type TouchCallback = (p: Point) => any;
+
+  export class Touch extends AbstractDispatcher {
+    private static _DISPATCHER_KEY = "__Plottable_Dispatcher_Touch";
+    private translator: _Util.ClientToSVGTranslator;
```

Comment:
I'll note I forgot to do that on `Dispatcher.Mouse` too. Does it make sense to fix it on this PR?